cpp/daal/BUILD

```diff
@@ -1,13 +1,40 @@
 package(default_visibility = ["//visibility:public"])
 load("@onedal//dev/bazel:daal.bzl",
     "daal_module",
-    "daal_depset",
     "daal_static_lib",
+    "daal_dynamic_lib",
     "daal_algorithms",
     "daal_generate_version",
     "daal_patch_kernel_defines",
 )
 
+daal_module(
+    name = "microvmlipp",
+    override_deps_lib_tag = True,
+    deps = [
+        "@micromkl//:vml_ipp",
+        # FIXME: Currently vml_ipp lib depends on TBB, but it shouldn't
```

Comment:
FIXME vs TODO - I think we should use one word in the library

Reply:
Ok